src/main/java/skyler/Task.java

```diff
@@ -0,0 +1,44 @@
+package skyler;
+
+/**
+ * Represents a generic task
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Creates a task object
+     *
+     * @param description Description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
```

Comment:
Minor issue but I think the brackets can be removed.
```suggestion
        return isDone ? "X" : " ";
```